yarn.lock

```diff
@@ -5,24 +5,34 @@
 "@babel/runtime@^7.0.0-beta.46":
   version "7.0.0-rc.2"
   resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.0.0-rc.2.tgz#63082909bd27c39f92c27c641f278389ad30a478"
-  integrity sha512-QugZtHk0uVHqHmgwNn2f2uOtTwXWPBBZVSH2EFRm9iw1fFmi5fm46ye6NSVSiXT9Ymejh7fKIw3riCv23SNtSQ==
```

Comment:
Upgrade yarn, and run `bench setup requirements`. You'll get a much smaller diff.

Reply:
Noted